indexing-service/src/main/java/io/druid/indexing/common/task/VersionConverterTask.java

```diff
@@ -84,17 +85,19 @@ private static VersionConverterTask createFromJson(
       @JsonProperty("dataSource") String dataSource,
       @JsonProperty("interval") Interval interval,
       @JsonProperty("segment") DataSegment segment,
-      @JsonProperty("indexSpec") IndexSpec indexSpec
+      @JsonProperty("indexSpec") IndexSpec indexSpec,
+      @JsonProperty("force") Boolean force
   )
   {
+    final boolean isForce = force == null ? false : force;
     if (id == null) {
       if (segment == null) {
-        return create(dataSource, interval);
+        return create(dataSource, interval, indexSpec, isForce);
       } else {
-        return create(segment);
+        return create(segment, indexSpec, isForce);
       }
     }
-    return new VersionConverterTask(id, groupId, dataSource, interval, segment, indexSpec);
+    return new VersionConverterTask(id, groupId, dataSource, interval, segment, indexSpec, isForce);
   }
 
   private VersionConverterTask(
```

Comment:
I wonder if it would make sense to rename this to be index-conversion task or something like that, given that we can now change things like bitmap and column compression, and this would not just be limited to version updates.